Makefile

```diff
@@ -67,3 +67,4 @@ endif
 test: i18n
```

Comment:
the `i18n` above compiles the templates, it should be removed if we have proper tests.

Reply:
What does this new approach improve on from the old approach? Don't they both just compile the templates? NOTE that this does **not** guarantee that the template can be rendered in any given language because parameter mismatches, message catalog syntax errors, etc can all cause runtime errors.

Reply:
Ah, these are testing 2 different things; `i18n` tests that the `.po` files are syntactically valid; it doesn't look at the `.html` files at all. The new test checks that the `.html` template files are syntactically valid. Marking this as won't fix.

(More specifically, the `i18n` step ensures that translators, who are editing the `.po` files, didn't make a mistake. The new test ensures that programmers, who are editing the `.html` template files, didn't make a mistake.)

Reply:
Hmm, If that is the case I'm questioning the value / appropriateness of running the `i18n` task as a test. I thought it did more than that, but it seems you are correct. It also illustrates the `i18n` in the test section isn't a very clear indication of what is being tested.

Reply:
We could put it in the lint section? I think that probably makes more sense for both of these, actually. Travis runs those, right?

Reply:
Separated into the _build_ step! We forget that there's such a thing as a build step when working in python sometimes :P https://github.com/internetarchive/openlibrary/pull/2930